2200031/14649.py

```diff
@@ -0,0 +1,38 @@
+T = int(input())
+
+for i in range(1, T+1):
+    num_hol = []
+    num_jjak = []
+    hol_sum = 0
+    jjak_sum = 0
+    N = 0
+    result = 0
+
+    num = list(map(int, input().split()))
+    
+    for j in range(len(num)):
+        if j%2 == 0:
+            num_hol.append(num[j])
+            hol_sum = sum(num_hol*2)
+        else:
+            num_jjak.append(num[j])
+            jjak_sum = sum(num_jjak)
```

Comment:
좋은 풀이네요! 👍 
```suggestion
    for j in range(len(num)):
        if j%2 == 0:
            num_hol.append(num[j])
        else:
            num_jjak.append(num[j])
    hol_sum = sum(num_hol*2)
    jjak_sum = sum(num_jjak)
```
`for` 문 밖에서 `sum` 연산을 해주면 연산하는 양을 많이 줄일 수 있습니다 😄